frontend/src/components/form/api/ApiWrapperCheckboxAppend.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="ec-api-checkbox-append mt-n1">
+    <!-- Retry/Cancel button if saving failed -->
```

Comment:
The Checkbox Append has now cancel button anymore? It's questionable, if we ever need a Checkbox in non-autosave mode. But shouldn't it have one, for consistency with the other input components?

Reply:
I think for an input that is a boolean, it just doesn't make sense to have this. I've shown this to multiple people and everyone thought it would remove the whole item, instead of resetting it to the previous state.